storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -134,6 +134,8 @@
 import clojure.lang.Keyword;
 import clojure.lang.RT;
 
+import static java.lang.Thread.currentThread;
```

Comment:
`Thread.currentThread()` is widely used so it's more readable than `currentThread()` for me.